src/app/pages/communication/cat-b/communication.cat-b.page.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { NavController } from '@ionic/angular';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+import { RouteByCategoryProvider } from '@providers/route-by-category/route-by-category';
+
+@Component({
+  selector: 'app-communication.cat-b',
```

Comment:
dont think selectors should have dots in them?

Reply:
were generated files, have replaced . with -